msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -421,6 +433,7 @@ void mergeConfiguration(PublicClientApplicationConfiguration config) {
         this.mLoggerConfiguration = config.mLoggerConfiguration == null ? this.mLoggerConfiguration : config.mLoggerConfiguration;
         this.webViewZoomControlsEnabled = config.webViewZoomControlsEnabled == null || config.webViewZoomControlsEnabled;
         this.webViewZoomEnabled = config.webViewZoomEnabled == null || config.webViewZoomEnabled;
+        this.powerOptCheckEnabled = config.powerOptCheckEnabled == null || config.powerOptCheckEnabled;
```

Comment:
So it seems like if they don't declare anything in config then the power opt check is enabled by default. Do we want that? Or should it be disabled by default?

Reply:
yes, given we removed `UsageStats.isAppInactive` check , I think we would be safer not to enable this by default, we can also some data from Teams to make a decision.

Reply:
@kreedula So you are going to update the code? It seems enabled by default at the moment

Reply:
@shahzaibj Apologies, edited the comment above